extensions/wikia/MyHome/ActivityFeedTag.php

```diff
@@ -25,34 +24,48 @@ function ActivityFeedTag_render($content, $attributes, $parser, $frame) {
 
 	$parameters = ActivityFeedHelper::parseParameters($attributes);
 
-	$tagid = str_replace('.', '_', uniqid('activitytag_', true));	//jQuery might have a problem with . in ID
+	$element =
+		renderElement( ActivityFeedHelper::getList( $parameters ), getStyle( $parameters ),
+			getSnippets( $parameters ) );
+	wfProfileOut( __METHOD__ );
+
+	return $element;
+}
+
+function renderElement($content, $style, $jsSnippets) {
+	return Html::element( 'div', [ 'style' => $style ], $content ) . $jsSnippets;
```

Comment:
I don't think it's appropriate to use sprintf here (it would look like `sprintf('%s%s', $first, $second);`).